Lesson 7 Task 4 - Text Length

jean-farinea opened the pull request:
1. Class TextLength created with method to calculate the text lenght
2. Test class created within parameterizedtest and provided sample texts

jean-farinea committed:
1. Class TextLength created with method to calculate the text lenght
2. Test class created within parameterizedtest and provided sample texts

strzeleckik commented on src/main/java/lesson7/task4/TextLength.java:
```diff
@@ -0,0 +1,10 @@
+package lesson7.task4;
+
+public class TextLength {
```
TextUtils maybe?

strzeleckik commented on src/main/java/lesson7/task4/TextLength.java:
```diff
@@ -0,0 +1,10 @@
+public class TextLength {
+
+    public static int calculateTextLength(String text) {
+        int textLength = text.length();
```
what if text is null? :)

strzeleckik reviewed (changes requested):
feedback to consider

jean-farinea committed:
1. Class name adjusted to TextUtils
2. Throwing null pointer exception at the method now.
3. Tests adjusted within methods

jean-farinea commented:
Changes done, please review.

strzeleckik commented on src/test/java/lesson7/task4/TextUtilsTest.java:
```diff
@@ -0,0 +1,26 @@
+        //when
+        int calculatedLength = TextUtils.calculateTextLength(text);
+        //then
+        Assertions.assertEquals(expectedLength, calculatedLength);
```
bad practice to check positive and negative scenario in one test

strzeleckik reviewed (changes requested):
feedback in comment

jean-farinea committed:
1. Test class adjusted to positive and negative scenario

jean-farinea commented:
Test adjusted, please review.

strzeleckik reviewed (approved):
ok
